compiler/rustc_mir_build/src/build/matches/test.rs

```diff
@@ -499,7 +499,7 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
     /// However, in some cases, the test may just not be relevant to candidate.
     /// For example, suppose we are testing whether `foo.x == 22`, but in one
     /// match arm we have `Foo { x: _, ... }`... in that case, the test for
-    /// what value `x` has has no particular relevance to this candidate. In
+    /// what value `x` has no particular relevance to this candidate. In
```

Comment:
Ah indeed. Hmm. How about:
```suggestion
    /// the value of `x` has no particular relevance to this candidate. In
```

Reply:
sounds good